index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="styles.css" rel="stylesheet" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+  </head>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <main>
+      <section class="Question-Box">
+        <h2 class="question">What property flips the axes in flexbox?</h2>
+        <button class="card_bookmark">🧠</button>
+            
+        </button>
+        <ul>
+          <li class="show answer">
+            <a href="#show answer">Show Answer</a>
+          </li>
+
+          <li class="tag">
+            #html
+          </li>
+
+          <li class="tag">
+            #flexbox
+          </li>
+
+          <li class="answer">
+            #css
+          </li>
+        </ul>
```

Comment:
The show answer element should not be part of the ul that contains all tags.

And semantically, this element should be a button and not an anchor tag.

We use anchor tags to navigate to other pages / within the same page.
For interactivity we use buttons. (Like clicking on something and showing/hiding other elements.

I recommend this HTML:

```suggestion
<button class="show answer">Show Answer</button>
<ul>
          <li class="tag">
            #html
          </li>
          <li class="tag">
            #flexbox
          </li>
         <li class="answer">
            #css
          </li>
 </ul>
```